packages/ui/.swcrc

```diff
@@ -8,5 +8,5 @@
   "module": {
     "type": "es6"
   },
-  "exclude": ["\\.stories\\.tsx?$", "src/samples", "src/helpers"],
-}
\ No newline at end of file
+  "exclude": ["src/samples", "src/helpers"],
+}
```

Comment:
@pmelab without this change I was getting
```
@custom/ui:test:integration: FAIL browser: chromium src/components/Organisms/ContentHub.stories.ts
@custom/ui:test:integration:   ● Test suite failed to run
@custom/ui:test:integration:
@custom/ui:test:integration:     cannot process file because it's ignored by .swcrc
```

Please let me know if it should be solved in a different way.